### ISSUE TITLE
The "help/more info" and "warning" icons under General Settings may not be screen reader accessible

### ISSUE THREAD

**Issue by ghostwords** [EFForg/privacybadger#3130]
Let's audit with screen reader. The circle question mark links should say something more helpful than just "link". The triangle exclamation icons should be screen reader accessible.
